### ISSUE TITLE
mg_classifier.sh not available?

### ISSUE THREAD

**Issue by DEsalasL** [GenomicaMicrob/mg_classifier#1]
Hi there, 
I have cloned the repository but have not been able to install the classifier as the directories do not contain the file mg_classifier.sh in the scripts directory. Would you please make it available or point me to the most updated instructions?
Many thanks

**Comment by GenomicaMicrob** [comment]
Yes, sorry for this, the script is almost ready and I will uploaded it as soon as possible.

Saludos

Bruno Gómez Gil R.
CIAD, A.C. Unidad Mazatlán
+52 (669) 989 8700 ext. 245
bruno@ciad.mx

**Comment by fcoaliaga** [comment]
Hi,

I've cloned the repository but I didn't find the script install.sh to finish the installation.
Can you tell me where I can get it?
Many thanks
